Modulo1-Fundamentos/6-HTML_e_CSS:_Forms,_Flexbox_e_Responsivo/6.1-HTML-&-CSS-Forms/exercicios/style.css

```
@@ -0,0 +1,101 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+#heder {
+    left: 0px;
+    top: 0px;
+    right: 0px;
+    width: 100%;
+    margin-right: 0px;
+    margin-left: 0px;
+    padding: 10px 0px;
+    border-bottom: 7px solid #2fc18c;
+    background-color: #036b52;
+    color: #fff;
+}
+
+#titulo {
+    margin-left: 28%;
+    display: inline-block;
+}
+
+#titulo h1 {
+    padding: 0px;
+    margin: 0px;
+    margin-bottom: 10px;
+}
+
+.image-45 {
+    display: inline-block;
+    margin-left: 40px;
+    width: 84px;
+    height: 50px;
+    max-width: 80%;
+}
+
+#harerMain{
+    margin-top: 0px;
+    padding: 20px;
+}
+
+#main{
+    color: rgb(93, 96, 99);
+    background-color: rgb(234, 237, 240);
+    font-family: "Helvetica", "Arial", sans-serif;
+    line-height: 1.3;
+}
+
+.form-header {
+    text-align: center;
+}
+
+.speaker-form {
+    background-color: rgb(246, 247, 248);
+    border: 1px solid rgb(214, 217, 220);
+    border-radius: 3px;
+    width: 80%;
+    padding: 50px;
+    margin: 20px auto;
+}
+
+.line-content {
+    margin-bottom: 20px;
+}
+
+.line-content input[type="text"], input[type="email"], input[type="date"] {
```

```suggestion
.line-content input[type="text"], .line-content input[type="email"], .line-content input[type="date"] {
```
Não sei se esse era o intuito, mas se for pra estilizar apenas os inputs de dentro do .line-content, é preciso repetir a declaração depois da vírgula. Considera esse código se essa for a ideia ;D